.gitignore

```diff
@@ -8,3 +8,4 @@ docs/.vuepress/public/assets/json/packages.json
 /packages/manager/tools/testcafe/config/**/*.json
 /reports
 .eslintcache
+.idea
```

Comment:
I think we shouldn't add `.idea` in the project `.gitignore`, you should add it in your default `.gitignore` (https://docs.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer & https://git-scm.com/docs/gitignore#_configuration)